jwst/resample/resample.py

```diff
@@ -251,6 +251,90 @@ def _get_intensity_scale(self, img):
         else:
             iscale = 1.0
         return iscale
+    
+    def resample_group(self, input_models, indices):
+        """Apply resample_many_to_many for one group
+        
+        Parameters
+        ----------
+        input_models : ModelLibrary
+
+        indices : list
+        """
+        output_model = self.blank_output
```

Comment:
I think output model handling would be a little more straightforward if you added the copy here instead of at the end of the function.
```suggestion
        output_model = self.blank_output.copy()
```